examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/DeprecatedOfferingFragment.kt

```diff
@@ -1,266 +0,0 @@
-package com.revenuecat.purchasetester
-
-import android.graphics.Color
-import android.os.Bundle
-import android.view.LayoutInflater
-import android.view.View
-import android.view.ViewGroup
-import android.widget.Toast
-import androidx.fragment.app.Fragment
-import androidx.navigation.fragment.findNavController
-import androidx.navigation.fragment.navArgs
-import androidx.recyclerview.widget.LinearLayoutManager
-import com.google.android.material.dialog.MaterialAlertDialogBuilder
-import com.google.android.material.transition.MaterialContainerTransform
-import com.revenuecat.purchases.CustomerInfo
-import com.revenuecat.purchases.Offerings
-import com.revenuecat.purchases.Package
-import com.revenuecat.purchases.Purchases
-import com.revenuecat.purchases.PurchasesError
-import com.revenuecat.purchases.UpgradeInfo
-import com.revenuecat.purchases.getCustomerInfoWith
-import com.revenuecat.purchases.getOfferingsWith
-import com.revenuecat.purchases.models.GoogleReplacementMode
-import com.revenuecat.purchases.models.StoreProduct
-import com.revenuecat.purchases.models.StoreTransaction
-import com.revenuecat.purchases.purchasePackageWith
-import com.revenuecat.purchases.purchaseProductWith
-import com.revenuecat.purchases_sample.R
-import com.revenuecat.purchases_sample.databinding.FragmentOfferingBinding
-
-@SuppressWarnings("TooManyFunctions")
-class DeprecatedOfferingFragment : Fragment(), DeprecatedPackageCardAdapter.PackageCardAdapterListener {
```

Comment:
This file was deleted because of the following snippet in `OverviewFragment`:  
```kotlin
val directions = OverviewFragmentDirections.actionOverviewFragmentToOfferingFragment(offering.identifier)
// uncomment line below and comment line above to test deprecated purchase methods
// val directions = OverviewFragmentDirections.
// actionOverviewFragmentToDeprecatedOfferingFragment(offering.identifier)
```  
Let me know if that was the wrong conclusion I took!

Reply:
No longer deleted.